### DISCUSSION TITLE
A few questions about PyVista

### DISCUSSION THREAD

**Discussion by Nikitap29** [pyvista/pyvista#1834] [Q&A] [answered]
Hi everyone!

I have a couple of questions about PyVista: 
1. In what units of measurement is the volume of the structure calculated? As far as I understand, in the same as the model itself (that is, if it is millimeters, then the volume is in cubic millimeters).
2. What algorithm underlies the compute_implicit_distance() function? How is the distance calculated approximately?

I will be glad to have a little consultation, I did not find such information in the documentation.

**Comment by banesullivan** [comment, marked as answer]
1. correct. Units are relative so that will match your source data approrpriately
2. `compute_implicit_distance()` is implement by the [`vtkImplicitPolyDataDistance` filter](https://vtk.org/doc/nightly/html/classvtkImplicitPolyDataDistance.html#details). I'll defer you to those docs and the paper that implemented this: https://www.vtkjournal.org/browse/publication/797

**Reply by Nikitap29** [reply]
Wonderful! Thank you for the answer!
